datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -216,12 +216,12 @@ def metadata(check):
             else:
                 reader._fieldnames = reader.fieldnames
 
-            for row in reader:
-
+            for line, row in enumerate(reader):
+                actual_line = line + 2  # First line is headers and line numbers start by 1 not 0
```

Comment:
(nit) Should we create a helper method here to avoid the prefix duplication (help consistency + easier to maintain) ?

Example:

```python
# helper method
echo_row_failure = lambda text: echo_failure('{}:{} {}'.format(current_check, actual_line, text))

# usage
echo_row_failure('{} Has the wrong amount of columns'.format(row['metric_name']))
```

Reply:
Yeah, thought about that but since you still have other params in the format I thought it was not really worth it